projects/control-service/projects/helm_charts/pipelines-control-service/values.yaml

```diff
@@ -1000,6 +1000,9 @@ dataJob:
       # In case of negative number it will be subtracted from the {{end_time}}.
       # If left blank, defaults to 0.
       endTimeOffsetSeconds: 0
+  deployer:
```

Comment:
What's the rationale behind using "deployer" instead of a construction similar to `dataJob.deployment.jobContainer.imagePullPolicy`?